src/utils/book-api.js

```diff
@@ -0,0 +1,43 @@
+import {useEffect, useState} from 'react';
+
+const books = {};
+
+export async function getBookData(isbn) {
+    if (books[isbn]) return books[isbn];
```

Comment:
Fetch will respect the cache headers from the server so we probably don't need to implement our own cache here.

Reply:
I didn't realize that it respected cache headers. I will remove this cache :+1: